src/js/components/receiving/ReceivingPage.jsx

```diff
@@ -91,11 +96,27 @@ class ReceivingPage extends Component {
 
       this.save(payload, this.nextPage);
     } else {
-      this.save({ ...formValues, receiptStatus: 'COMPLETED' }, () => {
-        this.setState({ completed: true });
-        const { requisition } = formValues;
-        window.location = `/openboxes/stockMovement/show/${requisition}`;
+      let isBinLocationChosen = true;
+      _.some(formValues.containers, (container) => {
```

Comment:
change it to:
`const isBinLocationChosen = !_.some(formValues.containers, container => _.some(container.shipmentItems, shipmentItem => _.isNull(shipmentItem.binLocation.id)));`